docs/services/slam/cartographer/_index.md

```diff
@@ -1,20 +1,25 @@
 ---
-title: "Cartographer Integrated Library"
+title: "Cartographer Modular Resource"
 linkTitle: "Cartographer"
 weight: 70
 type: "docs"
-description: "Configure a SLAM service with the Cartographer library."
+description: "Configure a SLAM service with the Cartographer modular resource."
 tags: ["slam", "services"]
+icon: "/services/img/icons/slam.svg"
 aliases:
   - "/services/slam/run-slam-cartographer"
 # SMEs: Kat, Jeremy
 ---
 
+services/img/icons/slam.svg
+
 [The Cartographer Project](https://github.com/cartographer-project) performs dense SLAM using LIDAR data.
 
+To easily integrate Cartographer into the Viam ecosystem, Viam provides [`viam-cartographer`](https://github.com/viamrobotics/viam-cartographer), a [modular resource](/program/extend/modular-resources/) which wraps Cartographer. This resource provides the `cartographer-module` binary for use directly on your SLAM-enabled robot.
```

Comment:
```suggestion
To easily integrate Cartographer into the Viam ecosystem, Viam provides [`viam-cartographer`](https://github.com/viamrobotics/viam-cartographer), a module which provides Cartographer as a
[modular resource](/program/extend/modular-resources/), a new model of SLAM service.
```
Particularly last part of sentnece is optional but I think the change to module is accurate? Basically the slam service itself is the resource 
A module can have multiple resources but this one is just the new service type

Reply:
Thank you! You're right!